_im-plugin/ism/policies.md

```diff
@@ -438,6 +439,120 @@ Parameter | Description | Type | Required
+                                            }
+                                        ]
+                                    },
+                                    {
+                                        "source_field": "total_quantity",
+                                        "metrics": [
+                                            {
+                                                "avg": {}
+                                            },
+                                            {
+                                                "max": {}
+                                            }
+                                        ]
+                                    }
+                                ]
+                            }
+                        }
+                    }
+                ],
+                "transitions": []
+            }
+        ]
+    }
+}
+````
+
+#### Field parameters
+
+The following field parameters are required:
+
+Field Parameter | Data Type | Description 
+:--- | :--- |:---
+description | String | The description name of the ISM policy. This field is optional.
+default_state | String | The default state to run for the ISM policy. Set the description to `rollup` for the ISM rollup policy.
+states | Nested list of objects | Define the states to be used in the ISM rollup policy.
+states.name | Nested list of objects | The description name of the state within the ISM policy. In this case, you will need to match the `default_state` description of "`rollup`."
+states.actions | Nested list of objects | Define the actions to be run by the state.
+states.transition | Nested list of objects | Define the conditions required to move on to the next state. When left blank, the policy completes its management of the index.
+states.actions.rollup | Nested list of objects | Define the actions taken during index rollup.
+states.actions.ism_rollup | Nested list of objects | Define the actions taken during ISM rollup.
+states.actions.rollup.ism_rollup.description | String | The description name of the `ism_rollup` field. This field is optional.
+states.actions.rollup.ism_rollup.target_index | String | The index to be used by the ISM rollup profile.
+states.actions.rollup.ism_rollup.page_size | Number | The number of indexes to be returned per page.
+states.actions.rollup.ism_rollup.dimensions.date_histogram.fixed_interval | Time | Define the interval period. Specify the fixed interval in milliseconds, seconds, minutes, hours, or days. For example, 60 minutes would be expressed as `60m`.
+states.actions.rollup.ism_rollup.dimensions.date_histogram.source_field | String | The name of the field to transform.
+states.actions.rollup.ism_rollup.dimensions.date_histogram.target_field | String | The target field that the newly transformed data is added to.
+states.actions.rollup.ism_rollup.dimensions.date_histogram.timezone | String | The time zone that triggers the rollup. Default is UTC.
```

Comment:
Is this the time zone that the rolled up documents are stored in?

Reply:
correct